dids/src/main/kotlin/web5/sdk/dids/ion/model/Models.kt

```diff
@@ -151,6 +169,11 @@ public data class OperationSuffixDataObject(
  */
 public typealias Commitment = String
 
+/**
+ * Type alias for reveal value.
+ */
+public typealias Reveal = String
```

Comment:
Nit: IMHO these comments are super obvious. I don't think they add value here and make the code less readable.

(https://github.com/TBD54566975/web5-kt/pull/77#discussion_r1354478198)